docs/source/user/settings.rst

```diff
@@ -160,6 +160,10 @@ the private network, the value should be the private IP of freeradius
 host like ``192.0.2.50``. If your freeradius is on a public network,
 please use the public IP of your freeradius instance.
 
+Similarly you can use subnet if freeradius hosted is on a variable IP.
+Example: ``198.168.0.0/24``. Use ``0.0.0.0/0`` to allow anyone to access
+freeradius (useful for development / testing).
+
```

Comment:
What about: 

```
You can also use subnets in case you need you can use subnet if freeradius hosted is on a variable IP, eg:

- ``198.168.0.0/24`` to allow the entire LAN.
- ``0.0.0.0/0`` to allow any address (useful for development / testing).